apps/api/src/report/constants/report-four-format.ts

```diff
@@ -0,0 +1,28 @@
+export const REPORT_FOUR_STEPS = [
```

Comment:
More of a clarification comment, are these in order of how the report is collecting the data for the report?

Reply:
Also, not 100% sure on this. Should this be in camel case?

Reply:
It looks great so far, I really like your idea with the order here.